auxano/templates/auxano/partnership.html

```diff
@@ -74,30 +74,47 @@ <h2 class="contact-title">Let's get your details</h2>
         <div class="col-lg-8">
             <form class="form-contact contact_form" action="" method="post" id="contactForm" novalidate="novalidate">
                 {% csrf_token %}
+                <div class="">
+                    {% if form.errors %}
+                    <div class="alert alert-danger alert-dismissible" role="alert">
+                        <div class="alert-message">
+                            {% for field in form %}
+                            {% for error in field.errors %}
+                            <span class="text-sm"><i class="fa fa-times"></i>&nbsp;&nbsp;{{ field.label }}: {{error|escape }}</span><br>
+                            {% endfor %}
+                            {% endfor %}
+
+                            {% for error in form.non_field_errors %}
+                            <i class="text-sm">{{ error|escape }}</i><br>
+                            {% endfor %}
+                        </div>
+                    </div>
+                    {% endif %}
+                </div>
                 <div class="row">
                     <div class="col-sm-6">
                         <div class="form-group">
-                            <input class="form-control valid" name="name" id="name" type="text" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Enter Your Name'" placeholder="Enter Your Name">
+                            <input class="form-control valid" name="{{form.name.name}}" id="name" type="text" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Enter Your Name'" placeholder="Enter Your Name">
                         </div>
                     </div>
                     <div class="col-sm-6">
                         <div class="form-group">
-                            <input class="form-control valid" name="email" id="email" type="email" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Enter Your Email Address'" placeholder="Your Email Address">
+                            <input class="form-control valid" name="{{form.email.name}}" id="email" type="email" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Enter Your Email Address'" placeholder="Your Email Address">
                         </div>
                     </div>
                     <div class="col-sm-6">
                         <div class="form-group">
-                            <input class="form-control valid" name="phone" id="phone" type="text" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Enter Your Phone Number'" placeholder="Enter Your Phone Number">
+                            <input class="form-control valid" name="{{form.phone_number.name}}" id="phone" type="text" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Enter Your Phone Number'" placeholder="Enter Your Phone Number">
```

Comment:
one of the reasons why the form is not submitting because the `name="phone_number"` is required but you are using `name="phone"`. Now I am using django generated form name